src/server/game/Battlegrounds/Battleground.cpp

```diff
@@ -1689,18 +1573,8 @@ bool Battleground::AddSpiritGuide(uint32 type, float x, float y, float z, float
     uint32 entry = (teamId == TEAM_ALLIANCE) ? BG_CREATURE_ENTRY_A_SPIRITGUIDE : BG_CREATURE_ENTRY_H_SPIRITGUIDE;
 
     if (Creature* creature = AddCreature(entry, type, x, y, z, o, teamId))
```

Comment:
This lane must go as well since you're creating a pointer that is not being used anywhere.

Reply:
fixed